### DISCUSSION TITLE
External Filters

### DISCUSSION THREAD

**Discussion by taalas** [rappasoft/laravel-livewire-tables#738] [Q&A]
Hi,

is there any way to use external inputs or selects to apply filtering for the datatable instance? Let's say I construct a select on another part of the page, can I use this to programmatically apply a filter?

**Comment by rappasoft** [comment]
Not right now, there's a possibility I may be able to expose some events to trigger from external sources though.

**Comment by taalas** [comment]
Ok, thanks for your quick reply! I think this would be an excellent addition, especially if wanting to differ from the current UI the component provides.

**Comment by rappasoft** [comment]
Added in upcoming 2.4: https://github.com/rappasoft/laravel-livewire-tables/commit/ad6203bb00779668a44271d7131a5a38715ac373#diff-3163eb29be61c0da7bd34efaacae80ad59ec628e829718c01e3a804c6363a16fR44

**Comment by inmanturbo** [comment]
@taalas as of the latest version you can now create filters and hide them from the default views. you can also trigger filters though events. This makes it possible for you to handle query filtering etc. from your custom selects the same way as you would with standard filters.
